File: Lógica de Programação Java e C#/5 - Arrays/Java/Exercícios de Fixação/Tabuada.java
class Tabuada{
	public static void main(String [] args){
		int[][] tabuada = new int [10][10];
		for(int a=1;a<tabuada.length;a++){
			System.out.println("");
			for(int b=1;b<tabuada.length;b++){
				tabuada[a][b]=a*b;
			System.out.println(a+" * "+b+" = "+tabuada[a][b]);
			}
		}
	}
}

File: Lógica de Programação Java e C#/5 - Arrays/C#/Exercício de Fixação/Tabuada.cs
class Tabuada
{
    static void Main()
    {
        int[][] tabuada = new int[10][];
        for (int a=0; a < tabuada.Length; a++)
        {
            tabuada[a] = new int [10];
            for (int b = 0; b < tabuada[a].Length; b++)
            {
                tabuada[a][b] = (a + 1) * (b + 1);
            }
        }
        for (int a = 0; a < tabuada.Length; a++)
        {
            System.Console.WriteLine("");
            for (int b = 0; b < tabuada[a].Length; b++)
            {
                System.Console.WriteLine((a + 1) + " * " + (b + 1) + " = " + tabuada[a][b]);
            }
        }
    }
}